### DISCUSSION TITLE
Unable to connect to mongodb atlas

### DISCUSSION THREAD

**Discussion by hasinireddybitla** [community/community#27101] [Actions]
<p>Unable to connect to mongodb atlas while running the ‘npm test’ through GitHub Actions.</p>
<p>Error: MongoNetworkError: failed to connect to server on first connect.</p>
<p>Any help is much appreciated. Thanks in advance!</p>

**Comment by otter-computer** [comment]
<p>Hi <a class="mention" href="/hasinireddybitla">@hasinireddybitla</a>, could you point us to an example workflow run where this is happening so we can take a look?</p>

**Comment by hasinireddybitla** [comment, marked as answer]
<p>Hi <a class="mention" href="/thomasshaped">@thomasshaped</a>, thanks for the reply. My error got resolved, its just whitelisting the ips in Mongodb Atlas. My bad, I missed on that.</p>

**Comment by sampadrout** [comment]
<p>Hi <a class="mention" href="/hasinireddybitla">@hasinireddybitla</a> can you please tell me how you wrote the steps in github action yml file to access mongodb remote atlas db?</p>
